src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -129,7 +134,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $input->getOption('router'), $env, null)) {
+        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $router, null)) {
```

Comment:
Thank you. Could also completely remove the `$timeout` parameter from the `createServerProcess()` method? It's not used anymore.